containers/Ideas/IdeaCreate.js

```diff
@@ -266,7 +275,7 @@ export const IdeaCreate = props => {
                 }}
                 onIconPress={() => setFieldValue('imageChecked', !values.imageChecked)}
                 checked={values.imageChecked}
-                image={values.image ? values.image.uri : (idea && idea.image)}
+                image={(values.image && values.image.uri) ? values.image.uri : (values.image && values.image)}
```

Comment:
This is needed because, if a new image was uploaded, it is an object with uri property, but if no new image has been uploaded, the value is set to idea.image (happens in getInitialValues) and that is already the uri of the image.